src/session.ts

```diff
@@ -191,6 +191,17 @@ export class SessionManager implements Middleware {
 
         if (this.sessionSettings.integratedConsole.suppressStartupBanner) {
             this.editorServicesArgs += "-StartupBanner '' ";
+        } else {
+            const packageJSON: any = require("../../package.json");
```

Comment:
Rather than importing this a second time (the first time is in `main.ts`) can you pass the [displayName](https://github.com/PowerShell/vscode-powershell/blob/master/package.json#L3) into this:

https://github.com/PowerShell/vscode-powershell/blob/26d3821a5a0f60afa3eaddcd7b4290d02108bb0e/src/main.ts#L135-L140

just like we do for the version... and then set it to a `HostName` property and use that in the startupBanner and for `-HostName ...` a few lines up?

Reply:
Sounds good.  I'll put that in after I get back from my run.  :-)

Reply:
Not tied to the `-HostName` name... could be shorter somehow...